examples/esi_try_example/Cargo.toml

```
@@ -0,0 +1,17 @@
+[package]
+name = "esi_try_example"
+version = "0.1.0"
+authors = []
+edition = "2018"
+# Remove this line if you want to be able to publish this crate as open source on crates.io.
+# Otherwise, `publish = false` prevents an accidental `cargo publish` from revealing private source.
+publish = false
```

This can inherit the properties from the workspace, as the other examples do.
```suggestion
version.workspace = true
authors.workspace = true
license.workspace = true
edition.workspace = true
publish = false
```